persistence-worker/main.py

```
@@ -83,71 +242,114 @@ async def run(self):
+                    else:
+                        print(f"[ERROR] Redis response error: {e}")
+                        await asyncio.sleep(5)
+                        continue
+                except Exception as e:
+                    print(f"[ERROR] Error reading messages: {e}")
+                    await asyncio.sleep(5)
+                    continue
+                try:
+
+                    if messages:
+                        for stream_name, entries in messages:
```

Ruff (0.14.11)

Loop control variable `stream_name` not used within loop body

Rename unused `stream_name` to `_stream_name`

(B007)